src/main/java/wooteco/subway/line/LineController.java

```diff
@@ -0,0 +1,54 @@
+package wooteco.subway.line;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        Line newLine = lineService.add(lineRequest.getName(), lineRequest.getColor());
+        LineResponse lineResponse = new LineResponse(newLine.getId(), newLine.getName(), newLine.getColor());
+        return ResponseEntity.created(URI.create("/lines/" + newLine.getId())).body(lineResponse);
+    }
+
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<Line> lines = lineService.lines();
+        List<LineResponse> lineResponses = lines.stream()
+            .map(it -> new LineResponse(it.getId(), it.getName(), it.getColor()))
+            .collect(Collectors.toList());
+        return ResponseEntity.ok().body(lineResponses);
+    }
+
+    @GetMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+        Line line = lineService.line(id);
+        LineResponse lineResponse = new LineResponse(line.getId(), line.getName(), line.getColor());
+        return ResponseEntity.ok().body(lineResponse);
+    }
+
+    @PutMapping(value = "/lines/{id}")
+    public ResponseEntity updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Object> updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
```

Raw 타입은 최대한 지양하고 제네릭을 명시해주는 것이 좋습니다 :)